### DISCUSSION TITLE
Coin Logo graph

### DISCUSSION THREAD

**Discussion by dianasta21** [taoteh1221/Slideshow_Crypto_Ticker#62] [Ideas]
Hello,

Great project, I was wondering if it's possible to add the Coin Logo graph beside the coin displayed?

Regards!

**Comment by taoteh1221** [comment]
Hello. Thanks for the compliments. This app is designed for small screens, so there isn't any room to add anything else next to the ticker price. We have to support at least 6 digits, plus the pair's ticker symbol, and price change arrow too.

**Comment by dianasta21** [comment]
Thanks for the quick reply!

I was thinking to add the Logo, not in the price line but on the left of the Coin Name which is the first line. I have a 3.5 inch screen and it looks like it will fit perfectly, attached image for reference ....


Regards.
![Slideshow_Crypto_Ticker 3 5inch](https://github.com/taoteh1221/Slideshow_Crypto_Ticker/assets/93454236/a83313f9-b1d9-46fb-8d39-95b157be1a59)

**Reply by taoteh1221** [reply]
2 problems doing that:

1) You disabled it showing the exchange name. If someone wants the exchange name to show (which is the default behavior), you can't show the logo.

2) We currently support UNLIMITED coins. You can add ANY coin on the supported exchanges. There is no RELIABLE way I'm aware of to automatically add a coin's logo. 

Generally when you develop software, keeping things simple keeps the app reliable under many different circumstances...in this case, adding many different coins. Hope this makes sense, but thanks for the feedback.

**Comment by dianasta21** [comment]
Thanks for the explanation, yes that makes sense!

Regards.

**Reply by taoteh1221** [reply]
No problem, I appreciate you taking the time to give feedback on the app. Thank you.
